Makefile

```diff
@@ -13,6 +14,7 @@ clean-pyc:
 
 develop: clean-pyc
 	pip install -e .[testing,docs]
+	nvm install
```

Comment:
Is nvm shipped with node.js? If not, could we check for it first?

Reply:
I don't think it's shipped with node.js. I just copied this additional instruction from the developing documentation here: https://github.com/wagtail/wagtail/blob/main/docs/contributing/developing.rst

Reply:
I think it might be better to just remove this line. Upon re-reading the developing docs I see that `nvm` is mentioned as an optional way to install the correct node version. I don't really feel checking for the existence of `nvm` should be part of this `Makefile`. All other tasks operate under the assumption the dev envoriment is set up correctly (e.g. a Python virtualenv exists and is of the correct python version, dependencies are installed etc.).